src/Simulation/TargetDefinitions/Decompositions/ApplyUncontrolledSAdj.qs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Intrinsic {
+    open Microsoft.Quantum.Math;
+
+    /// # Summary
+    /// Applies the -π/4 phase gate to a single qubit. Note that the Controlled functor
+    /// is not supported.
+    ///
+    /// # Description
+    /// \begin{align}
+    ///     SAdj \mathrel{:=}
+    ///     \begin{bmatrix}
+    ///         1 & 0 \\\\
+    ///         0 & -i
+    ///     \end{bmatrix}.
+    /// \end{align}
+    ///
+    /// # Input
+    /// ## qubit
+    /// Qubit to which the gate should be applied.
+    internal operation ApplyUncontrolledSAdj (qubit : Qubit) : Unit {
+         ApplyUncontrolledRz(-PI() / 2.0, qubit);
+    }
+}
```

Comment:
(This question can be very naïve, and non-applicable)
I see that
`ApplyUncontrolledS()` and 
`ApplyUncontrolledSAdj()` are implemented in different files, rather than the same file. Such an implementation makes an impression that there can be use cases when 
`ApplyUncontrolledS()` is used (and corresponding file is "included", speaking in C++ terminology), but the 
`ApplyUncontrolledSAdj()` is not (and corresponding file is _not_ included). And viced versa.

Is that really the case? (there can be such use cases) If yes, then OK. But if not then it can make sense to merge the two files together.

Reply:
Also when I look at "src/Simulation/TargetDefinitions/Decompositions/RxxFromSinglyControlled.qs", fragment 
```qsharp
    operation Rxx (theta : Double, qubit0 : Qubit, qubit1 : Qubit) : Unit is Adj + Ctl {
        . . .
        adjoint (...) {
            Rxx(-theta, qubit0, qubit1);
        }
```
I get an impression that the adjoint version calls the non-adjoint with the opposite sign for the theta. 

The same approach can be used for `ApplyUncontrolledS[Adj]` (if merging them is applicable).